esp/src/eclwatch/QueryTestWidget.js

```diff
@@ -85,8 +85,9 @@ define([
             var context = this;
             WsTopology.GetWsEclIFrameURL(type).then(function (response) {
                 var src = response + encodeURIComponent(context.params.QuerySetId + "/" + context.params.Id + (postfix ? postfix : ""));
+                var completeSrc = "http://" + dojoConfig.urlInfo.wseclhostandport + src;
```

Comment:
Replace "http" with location.protocol (so it switches between http and https as needed)